File: java/warmup-1/parrotTrouble.java
/* We have a loud talking parrot. The "hour" parameter is the current hour 
 * time in the range 0..23. We are in trouble if the parrot is talking and the 
 * hour is before 7 or after 20. Return true if we are in trouble.
 */
public boolean parrotTrouble(boolean talking, int hour) {
    return talking && (hour < 7 || hour > 20);
}

File: python/warmup-1/parrot_trouble.py
# We have a loud talking parrot. The "hour" parameter is the current hour 
# time in the range 0..23. We are in trouble if the parrot is talking and the 
# hour is before 7 or after 20. Return True if we are in trouble.
def parrot_trouble(talking, hour):
    return talking and (hour < 7 or hour > 20)
